openpype/hosts/fusion/hooks/pre_fusion_setup.py

```diff
@@ -14,101 +12,40 @@ class FusionPrelaunch(PreLaunchHook):
 
     def execute(self):
         # making sure python 3.6 is installed at provided path
-        py36_dir = self.launch_context.env.get("PYTHON36")
-        if not py36_dir:
-            raise ApplicationLaunchFailed(
-                "Required environment variable \"PYTHON36\" is not set."
-                "\n\nFusion implementation requires to have"
-                " installed Python 3.6"
-            )
-
-        py36_dir = os.path.normpath(py36_dir)
-        if not os.path.isdir(py36_dir):
+        py36_var = "FUSION16_PYTHON36_HOME"
```

Comment:
shouldn't this be renamed to something like "FUSION_PYTHON_HOME"?

Reply:
I agree - I think `FUSION_PYTHON3_HOME` would be the most sensible - it would even match the Fu18+ name for it too.